packages/core/src/document/editor.tsx

```diff
@@ -157,6 +157,7 @@ export function DocumentEditor({ id, pluginProps }: DocumentProps) {
             }
           }
         }}
+        allowChanges
```

Comment:
This fixes an issue with wrong handlern on `HotKeys` component: when typing Backspace or Delete checks the correct state for deleting empty plugins.